src/app/pages/settings/settings.component.ts

```diff
@@ -22,13 +22,28 @@ class SettingsComponent extends HTMLElement {
 	attributeChangedCallback(name: string, oldValue: string, newValue: string): void {
 		if ('data-modes' === name) {
 			let selectedMode = modeOfUseServiceInstance.getSelectedMode(JSON.parse(newValue));
+			this.openMainCategory(JSON.parse(newValue));
 			let elements = this.querySelectorAll(".c-settings__category");
 			const settings: string[] = Object.entries(JSON.parse(selectedMode))[0][1] as string[];
 			elements.forEach((element) => {
 				element.setAttribute('data-settings', JSON.stringify(settings));
 			});
 		}
 	}
+
+	openMainCategory = (modeOfUse: any): void => {
+		switch (modeOfUse.selectedMode) {
+			case 'facilePlus':
+				this.querySelector('app-text').setAttribute('data-is-main-category', 'true');
+				break;
+			case 'visionPlus':
+				this.querySelector('app-layout').setAttribute('data-is-main-category', 'true');
+				break;
+			default:
```

Comment:
C’est le même quer `facilePlus`, donc on pourrait joindre les deux : 

```suggestion
			case 'facilePlus':
			default:
```

En virant le premier cas, évidemment :)